mscore/shortcut.cpp

```diff
@@ -2776,8 +2776,8 @@ Shortcut Shortcut::_sc[] = {
        MsWidget::MAIN_WINDOW,
        STATE_NORMAL,
        "avsomr-hide-all",
-       QT_TRANSLATE_NOOP("action","Hide All Recognizing Glyphs"),
-       QT_TRANSLATE_NOOP("action","Toggle 'Hide All Recognizing Glyphs'"),
+       QT_TRANSLATE_NOOP("action","Hide All Recognition Glyphs"),
```

Comment:
How is that an improvment? Or what is a "Recognition Glyph"? Isn't it rather a "Regognized Glyph"?

Reply:
I was confused by this at first too, but the way I am interpreting it, the idea is that this option should hide both the recognized and unrecognized symbols together.  That is, "Hide glyphs that are there to show you whether the symbol was recognized or not".  In any case, that is what it seems to actually do :-).  But I agree the wording is unclear.  Maybe something like "Hide OMR hints", although "hints" isn't a great word either.

I would also say the option should be reversed, so the text says "Show" (with the default being false) instead of "Hide" (default true).  Just for consistency with the other View menu items.

Reply:
May be "Hide OMR Layer"?

Reply:
removed this code